src/cpp/desktop-mac/DockTileView.mm

```diff
@@ -24,7 +24,6 @@ - (id) init
 {
    if (self = [super init])
    {
-      showLabel_ = FALSE;
```

Comment:
Looks like this creates an empty if body -- should this function just be `return [super init]` now?